pom.xml

```diff
@@ -120,6 +115,27 @@
           <version>1.3</version>
           <scope>test</scope>
         </dependency>
+        <dependency> 
+          <groupId>org.jenkins-ci.plugins</groupId>
+          <artifactId>git</artifactId>
+          <version>2.6.5</version>
+          <classifier>tests</classifier>
+          <scope>test</scope>
+        </dependency>
+        <dependency>
+          <groupId>org.jenkins-ci.plugins.workflow</groupId>
+          <artifactId>workflow-multibranch</artifactId>
+          <version>2.11</version>
```

Comment:
2.11 will bring step-api with it. Upgrading workflow-aggregator would require bumping the minimum jenkins version and I'd rather get the fix out first